### PR TITLE
fix: cancel booking buttons design

### LINKED ISSUE #3702: cancel booking buttons are not responsive
image

### Steps to Reproduce
1) click on cancel on a booking

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes #3702 

Screenshot 2022-08-09 at 2 30 33 AM

### CODE REVIEW

File: apps/web/components/booking/CancelBooking.tsx [hunk 1]
```diff
@@ -57,19 +57,26 @@ export default function CancelBooking(props: Props) {
...
+                  onClick={() => router.push("/reschedule/" + booking?.uid)}>
                   {t("reschedule_this")}
                 </Button>
               </div>
             )}
-            <div className="w-full space-x-2 text-right">
-              <Button color="secondary" onClick={() => props.setIsCancellationMode(false)}>
+            <div className="mb-2 flex w-full space-x-2 text-right sm:mb-0 ">
+              <Button
+                className="rounded-md"
```
Comment (CarinaWolli) [inline apps/web/components/booking/CancelBooking.tsx:73]:
```suggestion
```

I think we should keep the old Button format for now. (https://github.com/calcom/cal.com/issues/3702#issuecomment-1208499417 this is already the new design)

File: apps/web/components/booking/CancelBooking.tsx [hunk 2]
```diff
@@ -57,19 +57,26 @@ export default function CancelBooking(props: Props) {
...
+                color="secondary"
+                onClick={() => props.setIsCancellationMode(false)}>
                 {t("nevermind")}
               </Button>
               <Button
+                className="flex grow justify-center rounded-md"
```
Comment (CarinaWolli) [inline apps/web/components/booking/CancelBooking.tsx:79]:
```suggestion
                className="flex grow justify-center"
```
Same here

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
Thank you for your contribution! Let's address the two comments I made before merging

**Review by CarinaWolli — APPROVED** [review]
Looks good now 👍

**Comment by Udit-takkar** [comment]
@CarinaWolli thanks for reviewing

**Pull request merged**